bromelia/lib/etsi_3gpp_s6a/messages.py

```diff
@@ -547,6 +547,130 @@ def __init__(self,
         DiameterRequest._load(self, locals())
 
 
+class ResetAnswer(DiameterAnswer):
+    """Implementation of Reset-Answer (RSA) command as per clause 7.2.16 of
+    ETSI TS 129 272 V15.4.0 (2018-07).
+
+    The Reset-Answer is indicated by the Command Code field set to 322 and
+    Command Flag's 'R' bit cleared.
+
+    Usage::
+
+        >>> from bromelia.lib.etsi_3gpp_s6a import RSA
+        >>> rsa = RSA()
+        >>> rsa
+        <Diameter Message: 322 [RSA] PXY, 16777251 [3GPP S6a], 5 AVP(s)>
```

Comment:
after applying reviews, make sure the "5 AVP(s)" will be correct in case following usage example